.github/workflows/ci.yml

```
@@ -154,9 +160,10 @@ jobs:
       cmake-args: -DBUILD_MQT_CORE_BENCHMARKS=ON -DBUILD_MQT_CORE_MLIR=ON -DBUILD_MQT_CORE_BINDINGS=ON
       clang-version: 20
       build-project: true
-      files-changed-only: true
+      files-changed-only: false
```

Set this back to `true` before merging. 

```suggestion
      files-changed-only: true
```